.github/actions/build-zeebe/action.yml

```diff
@@ -1,20 +1,11 @@
-# This action packages the complete Zeebe distribution artifacts. This includes the Go client,
-# zbctl, and the Zeebe distribution TAR ball. This excludes the Docker image. See the build-zeebe-docker
-# for that.
+# This action packages the complete Zeebe distribution artifacts.
+# This excludes the Docker image. See the build-platform-docker for that.
 
 ---
 name: Build Zeebe
 description: Builds & installs the complete Zeebe distribution
 
 inputs:
-  go:
-    description: If false, will not build zbctl; defaults to true
-    default: "true"
-    required: false
-  java:
```

Comment:
I figured there is no usage for this param anymore, given this only builds java now

Reply:
We need to adjust the workflow usage then

Reply:
the only ones using it were actually to go jobs which are already gone